contrib/net/http/option.go

```diff
@@ -206,3 +208,11 @@ func RTWithAnalyticsRate(rate float64) RoundTripperOption {
 		}
 	}
 }
+
+// RTWithIgnoreRequest holds the function to use for determining if the
+// outgoing HTTP request tracing should be skipped.
```

Comment:
Maybe a small change here to make it obvious that the request itself isn't being skipped, but tracing the request is?
```suggestion
// outgoing HTTP request should not be traced.
```

Reply:
Should I rephrase WithIgnoreRequest option above too?